src/react-components/src/components/02-molecules/Cards/EditionCard.stories.tsx

```diff
@@ -41,19 +43,19 @@ export const editionCardWithSomeEditionInfo = () => <EditionCard
   blockName=""
```

Comment:
The difference between the `editionCardWithSomeEditionInfo` and its following story isn't readily apparent to me other than that one includes double quotes around the Publisher information. What are these two stories intended to communicate?

Reply:
this was originally to show passing components, but it's better to consolidate them now since components are always passed.